docs/how-to/configure.md

```diff
@@ -202,13 +199,16 @@ This panel is where you connect your PMM server to your Percona Platform Account
     Your Percona Platform Account is separate from your PMM User account.
 
 ### Connect PMM to Percona Platform
+
+!!! alert alert-info ""
+    Disclaimer: Percona Platform is a Preview release and several functionalities are still under development or subject to change. As such, we recommend connecting PMM to Percona Platfotm only for testing purposes.
```

Comment:
1. Add a comma after Preview Release.
2. Add an article before the Percona Platform
3. Correct the spelling of Platform.

Reply:
this file showed up because I branched from "main" instead of 2.26.0. This file should not have been picked up so it now shows up as outdated and can be ignored.